### PR TITLE
Use github actions for running tests

### PR DESCRIPTION
**Pull request by JuliaSprenger**
This PR configures github actions to run pytest and coverall on the codebase.
Currently configured test systems are: {Ubuntu, MacOs, Windows} and {python 3.6, python 3.9}

Merging the PR would allow to move away from travis.com

### CODE REVIEW

File: .github/workflows/run_tests.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Display Python version
+      run: python -c "import sys; print(sys.version)"
+
+    - name: Install package
+      run: pip install .[tools,test]
+
+    - name: Lint with flake8
```
Comment (Slowblitz) [inline .github/workflows/run_tests.yml:35]: So we are using pep8 as coding style ? We should discuss it at some point

Comment (JuliaSprenger) [reply .github/workflows/run_tests.yml:35]: Yes, I think you also didn't like Black, so for now pep8 should be ok, no?

Comment (Slowblitz) [reply .github/workflows/run_tests.yml:35]: Yes seems good to me .

Comment (SylvainTakerkart) [reply .github/workflows/run_tests.yml:35]: +1 for PEP8, also coz it's well spread in neuro-softwares...

**Pull request merged**